en/reference/query-language-reference.html

```diff
@@ -406,6 +406,38 @@ <h2 id="where">where</h2>
         </td>
       </tr>
 
+      <tr>
+        <th>fuzzy</th>
+        <td>
+          <p id="fuzzy"><a href="https://en.wikipedia.org/wiki/Levenshtein_distance">Levensthein</a> edit distance search within an <a href='schema-reference.html#attribute'>attribute</a>.</p>
+<pre class="urlunencode" oncopy="">
+where%20myStringAttribute%20contains%20fuzzy(%22query%22)
+</pre>
+          <p>Annotations below are configuring <code>fuzzy</code>:</p>
+          <table class="table">
+            <thead>
+            <tr>
+              <th>Annotation</th>
+              <th>Effect</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+              <td><a href="#maxeditdistance">maxEditDistance</a></td>
+              <td>An inclusive upper bound of edit distance between query and string attribute (default is 2).</td>
+            </tr>
+            <tr>
+              <td><a href="#prefixlength">prefixLength</a></td>
+              <td>Number of characters that are considered frozen, so the fuzzy match will be performed only top of the suffix left. Default is 0 (i.e. <code>fuzzy</code> will match across whole query)</td>
```

Comment:
I am ensure about the meaning of this sentence: "... performed only top of the suffix left". Is 'top' the right word here?

Reply:
Yeah, good point, changed the wording